[docs] added ysql and ycql tabs for Akka

aishwarya24 opened the pull request:
Included YSQL and YCQL tabs for Akka persistence integration with YugabyteDB. 
Content for YSQL remaining.

ashetkar commented:
- The name of the heading on the left pane should just be `Akka Persistence` instead of `Akka Persistence Cassandra` since now we have a separate tab for YSQL and YCQL each.
- Also, once I clicked on the YSQL tab, I could not go back to/find the YCQL tab.
- The sub-title for YSQL tab could be `Akka Persistence R2DBC` and that for YCQL tab could be `Akka Persistence Cassandra`

aishwarya24 commented:
@ashetkar - I had just submitted like a first cut thinking we'll probably add more. But till we have YSQL tab , I have made YCQL the default one and  addressed all your comments. Thanks for your feedback. Let me know if we can merge this PR.

ashetkar reviewed (approved):
This looks good to merge. Thanks
